Bucketeer/Sources/Public/BKTError.swift

```diff
@@ -3,8 +3,11 @@ import Foundation
 public enum BKTError: Error, Equatable {
     case badRequest(message: String)
     case unauthorized(message: String)
+    case forbidden(message: String)
     case featureNotFound(message: String)
```

Comment:
Please rename it to `notFound`.

Reply:
Renamed https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/7f1e8a8f7115e3e91afaec54d07a145ee1602521